inc/utility.php

```diff
@@ -369,37 +369,52 @@ function cptui_products_sidebar() {
  */
 function cptui_newsletter_form() {
 	?>
-<!-- Begin MailChimp Signup Form -->
-<div id="mc_embed_signup">
-	<form action="//webdevstudios.us1.list-manage.com/subscribe/post?u=67169b098c99de702c897d63e&amp;id=9cb1c7472e" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
-		<div id="mc_embed_signup_scroll">
-			<p><strong><?php esc_html_e( 'Get email updates from pluginize.com about Custom Post Type UI', 'custom-post-type-ui' ); ?></strong></p>
-			<div class="mc-field-group">
-				<label for="mce-EMAIL"><?php esc_html_e( 'Email Address', 'custom-post-type-ui' ); ?></label>
-				<input tabindex="-1" type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL">
-			</div>
-			<div id="mce-responses" class="clear">
-				<div class="response" id="mce-error-response" style="display:none"></div>
-				<div class="response" id="mce-success-response" style="display:none"></div>
-			</div>    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
-			<div style="position: absolute; left: -5000px;" aria-hidden="true">
-				<input type="text" name="b_67169b098c99de702c897d63e_9cb1c7472e" tabindex="-1" value=""></div>
-			<div class="clear">
-				<input type="submit" value="<?php esc_attr_e( 'Subscribe', 'custom-post-type-ui' ); ?>" name="subscribe" id="mc-embedded-subscribe" class="button" tabindex="-1">
-			</div>
-		</div>
-	</form>
+
+<div class="email-octopus-form-wrapper">
+    <h2 class="email-octopus-heading">Subscribe to Pluginize.com</h2>
```

Comment:
Here is a screenshot comparing the previous version with the updated version of the Promo signup.

The previous version appears on the left, the updated version on the right.

![Screen Shot 2020-02-04 at 16 05 09 ](https://user-images.githubusercontent.com/5194588/73786989-56674200-4768-11ea-9425-af4d7b1fad20.png)

Despite the fact we were given the markup as a block, I don't see any reason we can't restore the text to 

> Get email updates from pluginize.com about Custom Post Type UI

unless this text update was a specific branding change (which I suspect it was not, but I could be wrong).

cc: @tw2113

Reply:
The text can absolutely be set back to the original. However, the gist Brad had provided had the updated title text, felt safer to carry the change over.

Reply:
Went ahead and updated header.